src/ui/screens/NotificationSettingsScreen.tsx

```diff
@@ -32,18 +63,36 @@ export function NotificationSettingsScreen({}: PropTypes) {
                 key={item.id}
                 title={item.label}
                 right={() => (
-                  <Switch value={item.selected} onValueChange={(subscribe) => toggleTopic({id: item.id, subscribe})} />
+                  <Switch
+                    value={item.selected}
+                    onValueChange={(subscribe) => toggleTopic({id: item.id, subscribe})}
+                    disabled={switchDisabled}
+                  />
                 )}
               />
             ))
           )}
           <Padder scale={1} />
           <Divider />
-        </View>
-        <View style={styles.infoContainer}>
-          <Icon name="information-outline" />
-          <Padder scale={1} />
-          <Text>Don't forget to enable notifications in your phone's settings</Text>
+          {!permissionAllowed(pushAuthorizationStatus) && (
+            <View style={styles.container}>
+              <View style={styles.infoContainer}>
+                <View
+                  style={{...globalStyles.rowContainer, ...globalStyles.alignCenter, ...globalStyles.justifyCenter}}>
+                  <Headline style={styles.headline}>Turn on notifications?</Headline>
+                </View>
+                <Subheading>
+                  <Caption style={{...globalStyles.textCenter}}>
```

Comment:
Same here, no need to spread and put it in an object again. You can just do `style={styleObject}`